src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -253,12 +253,11 @@ class HttpGoogleServicesDAO(
         //This can be used to auto-delete all objects next time the Google lifecycle manager runs (~every 24h).
         //More info: http://bit.ly/1WCYhhf and http://bit.ly/1Py6b6O
         val deleteEverythingRule = new Lifecycle.Rule()
-          .setAction(new Action().setType("delete"))
```

Comment:
Not only that, they failed to provide constants for the correct values in their client API jar.